src/index.js

```diff
@@ -0,0 +1,25 @@
+const { exec } = require('child_process')
+
+module.exports = function (pluginConfig, config, cb) {
+  exec('git tag', (err, stdout, stderr) => {
+    if (err) return cb(err)
+
+    const versionRegex = /^v((?:[0-9]+\.){2}[0-9]+)$/
+
+    const tags = stdout
+                  .trim()
+                  .split('\n')
+                  .map((t) => t.trim())
+                  .filter(t => versionRegex.test(t))
```

Comment:
In one line you're using the braces around `t` and in the next line you're omitting them. Please stick with one way.
Also renaming the variable `t` to `tag` should make things clearer.